### ISSUE TITLE
Unable to remove the start padding on line chart

### ISSUE THREAD

**Issue by EzekielWachira** [carlosgub/kotlinm-charts#4]
I'm trying to remove the space that seems to be a padding at the left side of the line chart but I'm unable to do so

```kt
  LineChart(
            lineChartData = lineData,
            modifier = Modifier.height(100.dp)
                .padding(start = 0.dp),
            xAxisLabel = {
            },

            animation = ChartAnimation.Sequenced(),
            yAxisLabel = {
            }
        )
```

![Screenshot_20230920_14060910](https://github.com/carlosgub/kotlinm-charts/assets/48563395/4fbbb959-3928-4b69-89c5-934becd0a4f8)

**Comment by carlosgub** [comment]
Hi,

I fix this in 1.0.4 (update to this new version)

<img width="919" alt="Screenshot 2023-09-20 at 2 58 14 PM" src="https://github.com/carlosgub/kotlinm-charts/assets/30916886/413bfa3a-a65d-4934-8ece-a86a8e043cbf">

In this new version if you don't pass the yAxisLabel, as you see in my image you remove the padding.

So modify your code as this

```
LineChart(
      lineChartData = lineData,
      modifier = Modifier.height(100.dp)
          .padding(start = 0.dp),
      xAxisLabel = {
      },
      animation = ChartAnimation.Sequenced()
)
```

**Comment by EzekielWachira** [comment]
I'm getting this: 

`Failed to resolve: com.carlosgub.kotlinm.charts:charts:1.0.4`
